db/migrate/20220810132120_create_likes.rb

```diff
@@ -1,8 +1,8 @@
 class CreateLikes < ActiveRecord::Migration[7.0]
   def change
     create_table :likes do |t|
-      t.integer :Author_id, foreign_key: true
-      t.integer :Post_id, foreign_key: true
+      t.integer :Author_id
+      t.integer :Post_id
```

Comment:
## [OPTIONAl]

- [ ] Kindly note that the name in convention is extremely important in Ruby on Rails, Only one name in convention error can mess up your code. I would kindly suggest you lowercase and separate words with `_` all of your columns in your tables. For example use `post_id` instead of `Post_id`, `Author_id` instead of `author_id`.......